File: dmytryshyn/Lab3/Lab3/Lab3/Classes/Ships/Implementations/LightWeightShip.cs
using MySecondPattern.Classes.Ships;
using MySecondPattern.Classes.Ships.Characteristics;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lab3.Classes.Ships.Implementations
{
    public class LightWeightShip : Ship
    {
        public LightWeightShip(int id, double fuel, ShipLimitsCharacteristics limitsCharacteristics) : base(id, fuel, limitsCharacteristics)
        {
        }
        public override string ToString()
        {
            return $"The ship {Id} is a lightweight ship.";
        }
    }
}

File: mukha/lab_3/PortManagementSystem/ships/light_weight_ship.py
from ships.ship import Ship

class LightWeightShip(Ship):
    def calculate_fuel(self):
        # Логіка розрахунку пального для легкого корабля
        self.fuel = self.capacity * 0.1  # Приклад логіки розрахунку пального

    def load_container(self, container):
        # Логіка завантаження контейнера на корабель
        pass
